MaterialLib/MPL/Utils/FormEigenTensor.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * File:   FormEigenTensor.h
+ *
+ * Created on July 29, 2019, 2:48 PM
+ */
+
+#pragma once
+
+#include <boost/variant/static_visitor.hpp>
+#include <Eigen/Dense>
+
+#include "MaterialLib/MPL/Property.h"
+#include "MaterialLib/MPL/PropertyType.h"
+
+namespace MaterialPropertyLib
+{
+template <int GlobalDim>
+class FormEigenTensor
```

Comment:
The class definition goes into cpp too, because it is not needed outside. Only the formEigenTensor() declaration (with explicit template instantiation declarations) remains in the header file.

Reply:
Thanks. Fixed accordingly by amending the last commit.